pmm-app/tests/qan_test.js

```diff
@@ -148,15 +150,83 @@ xScenario('Open the QAN Dashboard and check Details -> Tables', async (I, adminP
+  qanPage.selectFilter(qanPage.fields.filterCheckboxSelector);
+  qanPage.resetAllFilters();
+  I.dontSeeCheckboxIsChecked(qanPage.fields.filterCheckboxSelector);
+});
+
+Scenario('Open the QAN Dashboard, add a column and make sure that this option not available anymore @new-qan', async (I, qanPage) => {
+  qanPage.addColumn('Bytes Sent');
+  qanPage.verifyColumnIsNotAvailable('Bytes Sent');
+});
+
+Scenario('Open the QAN Dashboard and verify that hovering over a time metric displays a tooltip with a graph @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_TIME_COLUMN_NUMBER = 3;
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_TIME_COLUMN_NUMBER);
+  I.seeElement(qanPage.elements.latencyChart);
+});
+
+Scenario('Open the QAN Dashboard and verify that hovering over a non-time metric displays a tooltip without a graph @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_COUNT_COLUMN_NUMBER = 2;
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+  I.dontSeeElement(qanPage.elements.latencyChart);
+});
+
+Scenario('Open the QAN Dashboard and verify that the metric value matches the "Per sec" value in the tooltip. @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_COUNT_COLUMN_NUMBER = 2;
+  const queryCountColumnSelector = qanPage.summarizeLocator(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+  let qpsMetricValue = await I.grabTextFrom(queryCountColumnSelector);
+  let qpsTooltipValue = await I.grabTextFrom('[data-qa="metrics-list"] [data-qa="qps"] span');
+  assert.equal(qpsMetricValue.replace(/[^0-9.]/g,""), qpsTooltipValue.replace(/[^0-9.]/g,""));
+});
+
+Scenario('Open the QAN Dashboard and check that changing the time range clears the selected row. @new-qan', async (I, qanPage, adminPage) => {
+  I.waitForElement(qanPage.elements.tableRowSelector, 30);
+  I.forceClick(qanPage.elements.tableRowSelector);
+  adminPage.applyTimeRange('Last 3 hours');
+  I.dontSeeElement(qanPage.elements.selectedOverviewRow);
+  I.dontSeeElement(qanPage.elements.detailsSection);
+});
+
+Scenario('Open the QAN Dashboard and check that changing the time range resets current page to the first. @new-qan', async (I, qanPage, adminPage) => {
+  qanPage.paginationGoTo(2);
+  adminPage.applyTimeRange('Last 3 hours');
+  qanPage.waitForResponsePath('/v0/qan/GetReport');
+  qanPage.verifySelectedPageIs(1);
+});
+
+Scenario('Open the QAN Dashboard and check that changing the time range updates the overview table and URL. @new-qan', async (I, qanPage, adminPage) => {
+  I.amOnPage(qanPage.url + '?from=now-12h&to=now');
+  qanPage.waitForQANPageLoaded();
+  I.seeInCurrentUrl('from=now-12h&to=now');
+  adminPage.applyTimeRange('Last 3 hours');
+  qanPage.waitForResponsePath('/v0/qan/GetReport');
+  qanPage.waitForQANPageLoaded();
+  I.seeInCurrentUrl('from=now-3h&to=now');
```

Comment:
from=now-12h&to=now and from=now-3h&to=now could be stored as variables on top of a test
  /v0/qan/GetReport path  should be called from pageObj